Cli 1.6 docs follow-up (#1289)
astronomer/docs: astro/cli/astro-dev-object-import.md

---
sidebar_label: "astro dev object import"
title: "astro dev object import"
id: astro-dev-object-import
description: Reference documentation for astro dev object import
---

> Import Ariflow variables, connections, and pools to a locally running environment from a YAML configuration file.

Reviewer: ```suggestion
Import Airflow variables, connections, and pools to a locally running environment from a YAML configuration file.
```

> Import Airflow variables, connections, and pools to a locally running environment from a YAML configuration file.

Reviewer 2: ```suggestion
Import Airflow variables, connections, and pools from a configuration file to a locally running Airflow environment. When you run this command, the Airflow objects you import from your specified file will be saved to your local metadata database and appear in the Airflow UI of your local Airflow environment.
```

Author: I don't know how the second sentence adds to the understanding of what the command does. I would assume that someone looking to import these objects already knows how they're stored and shown in the Airflow UI. We're not describing the command here, but rather the behavior of variables themselves.

## Usage 

After starting your local Airflow environment with `astro dev start`, run:

```sh

> astro dev object import [flags]

Reviewer 2: ```suggestion
astro dev object import
```

```

By default, the command imports all variables, connections, and pools from `airflow_settings.yaml` to your project. You do not need to restart your environment for these changes to take effect. 

## Options

| Option              | Description                                                                                                        | Possible Values             |
| ------------------- | ------------------------------------------------------------------------------------------------------------------ | --------------------------- |
| `-c`,`--connections` | Import connections from a given local file | ``|                                                                      
| `-p`,`--pools`            | Import pools from a given local file | ``                 |
| `-s`,`--settings-file`            | Location of the file from which to import Airflow objects. The default file path is `~/.airflow_settings.yaml`.                                 | Any valid filepath              |
| `-v`,`--variables`            | Import variables from a given local file | ``                 |

> 

Reviewer 2: ```suggestion
```

## Examples 

```sh
astro dev object import --pools 
# Imports pools from `airflow_settings.yaml` to a locally running Airflow environment

astro dev object import --settingsfile="myairflowobjects.yaml"
# Imports all Airflow objects from `myairflowobjects.yaml` to a locally running Airflow environment

> ```

Reviewer 2: Related commands
